### PR TITLE
Fix circleci config

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -161,7 +161,7 @@ jobs:
             export PATH=${HOME}/node-v12.16.3-linux-x64/bin:${PATH}
             export PATH=${HOME}/.daml/bin:${PATH}
             export PATH=${HOME}/.yarn/bin:${PATH}
-            make buildui
+            make build
             yarn install
             yarn build
```
Comment (richardkapolnai-da) [inline .circleci/config.yml:166]: isn't this redundant?

Comment (gaborh-da) [reply .circleci/config.yml:166]: I think you have to call both (install + build)

Comment (richardkapolnai-da) [reply .circleci/config.yml:166]: But `make build` does that already, doesn't it?

Comment (gaborh-da) [reply .circleci/config.yml]: I think you are right, I removed `yarn install`.

Comment (richardkapolnai-da) [reply .circleci/config.yml]: sorry for writing one-by-one, but we also have a `make` goal for `yarn build`, right?

**Review by richardkapolnai-da — APPROVED** [review]

**Pull request merged**